src/components/Header.jsx

```diff
@@ -2,10 +2,12 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 
+import 'scss/style.scss';
```

Comment:
Our SCSS files will live in the same folder with their corresponding components. Please relocate and rename. Thanks

Reply:
This was just a sample. We will put that in its folder